src/components/common/PageHero.tsx

```diff
@@ -46,6 +46,10 @@ const Description = styled.div`
   p:last-child {
     margin-bottom: 0;
   }
+
+  a {
+    text-decoration: underline;
```

Comment:
Could we remove the underline on hover? It's a nice a11y practice to have a clear visual change on hover.

Reply:
Done! thanks for the suggestion